src/schemas/compare.py

```
@@ -31,6 +31,18 @@ class CompareRequest(BaseModel):
     country_id_1: int  # 기준 국가 ID (예: 1)
     country_id_2: int  # 비교 국가 ID (예: 2)
 
+    @field_validator("query")
+    def check_query(cls, v):
+        if not v or not v.strip():
+            raise ValueError("질문을 입력해주세요.")
```

Ruff (0.15.4)

[warning] Avoid specifying long messages outside the exception class

(TRY003)